src/fxController/Registration.java

```diff
@@ -1 +1,107 @@
- 
+ package fxController;
+
+import hibernate.Hibernate;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.TextField;
+import javafx.stage.Stage;
+import model.Driver;
+import model.Manager;
+
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Persistence;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Objects;
+import java.util.ResourceBundle;
+
+public class Registration implements Initializable {
+    @FXML
+    public TextField nameField;
+    @FXML
+    public TextField surnameField;
+    @FXML
+    public TextField usernameField;
+    @FXML
+    public TextField emailField;
+    @FXML
+    public TextField passwordField;
+    @FXML
+    public TextField repeatPasswordField;
+    @FXML
+    public RadioButton driverRadio;
+    @FXML
+    public RadioButton managerRadio;
+
+    EntityManagerFactory entityManagerFactory = Persistence.createEntityManagerFactory("CargoTransportationsSystem");
+    Hibernate hibernate = new Hibernate(entityManagerFactory);
+
+    public void driverAccount() {
+        managerRadio.setSelected(!driverRadio.isSelected());
+    }
+    public void managerAccount() {
+        driverRadio.setSelected(!managerRadio.isSelected());
+    }
+
+    public void createNewAccount() throws IOException {
+        StringBuilder errors = new StringBuilder();
+
+        if ((nameField.getText().trim().isEmpty()) || (surnameField.getText().trim().isEmpty()) || (usernameField.getText().trim().isEmpty()) ||
```

Comment:
if sąlyga per ilga. Galima sukurti atskirą funkciją, kuri grąžintų klaidos pranešimus, pvz.: "email field can not be empty".

Reply:
per daug if sąlygų. Jos taip pat gali būti kviečiamos iš vienos funkcijos. Reikėtų skaidyti kodą į validatorius, taip vėliau pernaudojant kodą.